server/lib/validation.js

```diff
@@ -8,29 +8,35 @@
 
 const joi = require('joi');
 
-module.exports = {
-  PATTERNS: {
-    BROKER: /^[0-9a-z-]+$/,
-    CLIENT_ID: /^[0-9a-f]{16}/,
-    CONTEXT: /^[0-9a-z_-]+$/,
-    ENTRYPOINT: /^[\w.:-]+$/,
-    EVENT_TYPE: /^[\w\s.:-]+$/, // the space is to allow for error contexts that contain spaces, e.g., `error.unknown context.auth.108`
-    EXPERIMENT: /^[\w.-]+$/,
-    MIGRATION: /^(sync11|amo|none)$/,
-    SERVICE: /^(sync|content-server|none|[0-9a-f]{16})$/,
-    UNIQUE_USER_ID: /^[0-9a-z-]{36}$/
-  },
+const PATTERNS = {
+  BASE64: /^[A-Za-z0-9\/+]+={0,2}$/,
```

Comment:
I know this regex was here before but I only just noticed: isn't it wrong? Base 64 strings come in 4-character chunks, so there can be up to 3 `=` characters, no? Shouldn't it be `={0,3}` or have I gone completely mental?